modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -84,19 +84,49 @@ IBM Power platform:
 | 120 ^[4]^
 | 1
 
-| Worker
+| Compute
 | 16
 | 64
 | gp2
 | 120
-| 3/25/250/500 ^[5]^
+| 2 to 100 ^[5]^
 
 |===
 [.small]
 --
-1. io1 disks with 120 / 3 IOPS per GB are used for master/etcd nodes as etcd is I/O intensive and latency sensitive.
+1. io1 disks with 120 / 10 IOPS per GiB are used for control plane/etcd nodes as etcd is I/O intensive and latency sensitive.
 2. Infra nodes are used to host Monitoring, Ingress, and Registry components to ensure they have enough resources to run at large scale.
 3. Workload node is dedicated to run performance and scalability workload generators.
 4. Larger disk size is used so that there is enough space to store the large amounts of data that is collected during the performance and scalability test run.
-5. Cluster is scaled in iterations and performance and scalability tests are executed at the specified node counts.
+5. Cluster is scaled in iterations.
+--
+
+== {ibmzProductName} platform:
+
+[options="header",cols="6*"]
+|===
+| Node |vCPU ^[4]^ |RAM(GiB)^[5]^|Disk type|Disk size(GiB)/IOS |Count
+
+| Control plane/etcd ^[1,2]^
+| 8
+| 32
+| ds8k
+| 300 / LCU 1
+| 3
+
+| Compute ^[1,3]^
+| 8
+| 32
+| ds8k
+| 150 / LCU 2
+| 4 nodes (scaled to 100/250/500 pods per node) 
+
+|===
+[.small]
 --
+1. Nodes are distributed between two logical control units (LCUs) to optimize disk I/O load of the control plane/etcd nodes as etcd is I/O intensive and latency sensitive. Etcd I/O demand should not interfere with other workloads.
+2. Four compute nodes are used for the tests running several iterations with 100/250/500 pods at the same time. First, we used pods idling to evaluate if pods can be instanced. Next, we used a network and CPU demanding client/server workload to evaluate the stability of the system under stress. Client and server pods were pairwise deployed and each pair was spread over two compute nodes.
```

Comment:
```suggestion
2. Four compute nodes are used for the tests running several iterations with 100/250/500 pods at the same time. First, idling pods were used to evaluate if pods can be instanced. Next, a network and CPU demanding client/server workload were used to evaluate the stability of the system under stress. Client and server pods were pairwise deployed and each pair was spread over two compute nodes.
```

Reply:
Suggestion, remove unnecessary pronouns ("we"), since this is consistent with the first and last sentences.